### PR TITLE
Add updates for quick launch partner

### PR DESCRIPTION
**Pull request by rmshree**
### What does this PR do? What is the motivation?
Updating the Metric Streaming guide to reflect the new "Quick Launch Partner" flow releasing on Oct 17.

### Merge instructions

- [ ] Please merge after Oct 17, 2023

### CODE REVIEW

**Comment by drichards-87** [comment]
Created DOCS-6437 for the editorial review.

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 1]
```diff
@@ -142,9 +131,7 @@ If you set streaming up with [CloudFormation](?tab=cloudformation#installation):
 
 If you set streaming up through the [AWS Console](?tab=awsconsole#installation):
 1. Delete the CloudWatch Metric Stream linked to your delivery stream.
-2. Delete the Kinesis Data Firehose delivery stream that delivered metrics to Datadog.
-3. Delete the backup S3 bucket for failed messages linked to the Firehose.
-4. Delete the IAM roles associated with the stream and all other resources that were created while setting up the stream.
+2. Delete the S3 Bucket, Firehose, IAM roles associated with the stream and all other resources that were created while setting up the stream.
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:134]: Just to confirm, this isn't saying to also delete "all other resources that were created while setting up the stream", right? That's how I first read the sentence but now I'm not sure if that's correct. Are we only talking about deleting roles here?

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 2]
```diff
@@ -130,9 +113,15 @@ Once you see the Metric Stream resource has been successfully created, wait five
 [2]: https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#metric-streams:streams/create
 [3]: https://github.com/DataDog/cloudformation-template/blob/master/aws_streams/streams_single_region.yaml#L168-L249
 [4]: https://app.datadoghq.com/integrations/amazon-web-services
+[5]: https://docs.datadoghq.com/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/#cross-account-metric-streaming
 {{% /tab %}}
 {{< /tabs >}}
 
+### Cross account metric streaming
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:120]:
```suggestion
### Cross-account metric streaming
```

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 3]
```diff
@@ -130,9 +113,15 @@ Once you see the Metric Stream resource has been successfully created, wait five
...
+Cross-account metric streaming is an option you can use to include metrics that span across multiple AWS accounts within an AWS region in a single Metric Stream. This helps to reduce the number of streams needed to collect metrics for a common destination. To do this, you will need to [connect your source accounts][5] with your monitoring account and enable Cross-account streaming to Datadog in your AWS monitoring account.
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:121]:
```suggestion
Use cross-account metric streaming to include metrics in a single Metric Stream that spans across multiple AWS accounts within an AWS region. This helps to reduce the number of streams needed to collect metrics for a common destination. To do this, [connect your source accounts][5] with your monitoring account and enable Cross-account streaming to Datadog in your AWS monitoring account.
```

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 4]
```diff
@@ -130,9 +113,15 @@ Once you see the Metric Stream resource has been successfully created, wait five
...
+
+**Note:** To collect custom tags and other metadata for all your streamed metrics, integrate your source accounts with Datadog.
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:123]:
```suggestion
**Note:** To collect custom tags and other metadata for your streamed metrics, integrate your source accounts with Datadog.
```

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 5]
```diff
@@ -93,28 +91,13 @@ Once the stack is successfully created, wait five minutes for Datadog to recogni
  
 If you want to set up metric streams using the AWS Console, follow these steps for each AWS region.
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:92]:
```suggestion
To set up metric streams using the AWS Console, create a [CloudWatch Metric Stream][2] for each AWS region.
```

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 6]
```diff
@@ -93,28 +91,13 @@ Once the stack is successfully created, wait five minutes for Datadog to recogni
...
+   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/metric-stream-partner-setup.png" alt="Cloudwatch metric stream quick partner setup" responsive="true" style="width:60%;">}}
+3. Choose the Datadog site to which you want to stream metrics and enter your [Datadog API key][1].
+4. Choose whether you want to stream all CloudWatch metrics, or only specific namespaces. You can optionally also exclude specific metrics. If you are in a Monitoring Account, you can also choose to enable [Cross-account streaming][5].
+   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/metric-stream-namespace-filter.png" alt="Cloudwatch metric stream" responsive="true" style="width:60%;">}}
+5. Under **Add additional statistics**, include the AWS percentile metrics to send to Datadog. See the [CloudFormation template][3] for a list of the percentile metrics Datadog supports through polling.
    {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/percentiles.png" alt="Percentiles" responsive="true" style="width:60%;">}}
 8. Name your metric stream.
 9. Click **Create metric stream**.
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:103]:
```suggestion
5. Assign a name to your metric stream.
6. Click **Create metric stream**.
```

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md [hunk 7]
```diff
@@ -93,28 +91,13 @@ Once the stack is successfully created, wait five minutes for Datadog to recogni
...
+1. Create your [CloudWatch Metric Stream][2] with the following steps:
+2. Choose the "Quick AWS Partner Setup" and select "Datadog" as the AWS Partner destination from the dropdown.
+   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/metric-stream-partner-setup.png" alt="Cloudwatch metric stream quick partner setup" responsive="true" style="width:60%;">}}
+3. Choose the Datadog site to which you want to stream metrics and enter your [Datadog API key][1].
+4. Choose whether you want to stream all CloudWatch metrics, or only specific namespaces. You can optionally also exclude specific metrics. If you are in a Monitoring Account, you can also choose to enable [Cross-account streaming][5].
+   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/metric-stream-namespace-filter.png" alt="Cloudwatch metric stream" responsive="true" style="width:60%;">}}
+5. Under **Add additional statistics**, include the AWS percentile metrics to send to Datadog. See the [CloudFormation template][3] for a list of the percentile metrics Datadog supports through polling.
```
Comment (drichards-87) [inline content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:100]:
```suggestion
1. Choose the **Quick AWS Partner Setup** and select **Datadog** as the AWS Partner destination from the dropdown menu.
   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/metric-stream-partner-setup.png" alt="Cloudwatch metric stream quick partner setup" responsive="true" style="width:60%;">}}
2. Choose the Datadog site to which you want to stream metrics and enter your [Datadog API key][1].
3. Choose whether you want to stream all CloudWatch metrics, or only specific namespaces. You also have the option to exclude specific metrics. If you are in a Monitoring Account, you can also choose to enable [Cross-account streaming][5].
   {{< img src="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/metric-stream-namespace-filter.png" alt="Cloudwatch metric stream" responsive="true" style="width:60%;">}}
4. Under **Add additional statistics**, include the AWS percentile metrics to send to Datadog. See the [CloudFormation template][3] for a list of the percentile metrics Datadog supports through polling.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Added some small suggestions from Docs. Also, just FYI, there appears to be a merge conflict in the PR.

File: content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md (hunk 1 above)
Comment (rmshree) [reply content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md:134]: You got it right - it does say to delete all resources that were created while setting up the stream! Thanks!

**Pull request merged**